central/pruning/pruning.go

```diff
@@ -547,6 +555,25 @@ func (g *garbageCollectorImpl) collectImages(config *storage.PrivateConfig) {
 	}
 }
 
+func (g *garbageCollectorImpl) collectReportHistory(config *storage.PrivateConfig) {
+	reportHistoryRetentionConfig := config.GetReportRetentionConfig().GetHistoryRetentionDurationDays()
+	// Check to see if enough time has elapsed to run report history pruning again
+	reportHistoryRetentionDays := 24 * time.Hour * time.Duration(reportHistoryRetentionConfig)
+	if lastReportPruneTime.Add(reportHistoryRetentionDays).After(time.Now()) {
+		// pruning if it's been at least report retention days since last run
+		return
+	}
```

Comment:
I don't think the collection will ever happen. `lastReportPruneTime` is unset; therefore, `lastReportPruneTime.Add(reportHistoryRetentionDays).After(time.Now())` is always false.

Reply:
Yes, I will fix the condition. I was going to send another pr for unit tests but will add them to this one